tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -26,6 +25,14 @@ Verify Pachyderm Can Be Installed Using OpenShift Console
     Wait Until Status Is Running
     Go To RHODS Dashboard
     Verify Service Is Enabled    Pachyderm
+    Launch JupyterHub Spawner From Dashboard
+    Wait Until Page Contains Element    xpath://input[@name="Standard Data Science"]
+    Wait Until Element Is Enabled       xpath://input[@name="Standard Data Science"]    timeout=10
+    Spawn Notebook With Arguments  image=s2i-generic-data-science-notebook
+    Create Pachyderm Pipeline Using JupyterLab
```

Comment:
I think it would be cleaner to keep the pipeline creation in another TC. You could follow the example of `minimal-tensorflow-test.robot` or `640_rhoam_installation` test suites.

Reply:
For Pipeline creation we need to have pachyderm installed and it's namespace has been created so if we move pipeline creation to another test-case then we need to put the installation and namespace creation portion into a keyword and as done in 640_rhoam_installation we need to put it in Setup keyword. But I don't think that if we are doing it in a single test-case will create a mess.

Reply:
I think the structure propsoed by @jgarciao and used in minimal-tensorflow-test.robot or 640_rhoam_installation makes the test suite cleaner and it splits logically two different test cases. I feel it would be a better approach, but it is just my opinion. if you feel this is good okay, but at least you should change the test case title

Reply:
Done !